pkg/cmd/pkg/package_uninstall.go

```diff
@@ -13,8 +27,69 @@ func newCmdPackageUninstall(ctx api.Context) *cobra.Command {
 		Use:   "uninstall <package-name>",
 		Short: "Uninstall a package",
 		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			appIDSet := cmd.Flags().Changed("app-id")
```

Comment:
It shouldn't be need to check for `Changed`, an empty appID can be considered as the option not being passed.